server/info.go

```diff
@@ -658,6 +661,35 @@ func (s *infoServer) mergeAdditionalInfoToTxs(txs []*types.Transaction, receipts
 		tx.TxFee = txFeeInHydro.String()
 
 		receiptIndex++
+
+		// write external contract data to database
+		if strings.ToLower(tx.To) == strings.ToLower(cfg.ParamsContractAddr) && tx.Status == 1 && decoded != nil {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA6005: should use strings.EqualFold instead (staticcheck)